MailAlert.php

```diff
@@ -74,18 +82,19 @@ public static function customerHasNotification($id_customer, $id_product, $id_pr
 			SELECT *
 			FROM `' . _DB_PREFIX_ . self::$definition['table'] . '`
 			WHERE ' . $where . '
+			AND deleted = 0
 			AND `id_product` = ' . (int) $id_product . '
 			AND `id_product_attribute` = ' . (int) $id_product_attribute . '
 			AND `id_shop` = ' . (int) $id_shop;
 
         return count(Db::getInstance((bool) _PS_USE_SQL_SLAVE_)->executeS($sql));
     }
 
-    public static function deleteAlert($id_customer, $customer_email, $id_product, $id_product_attribute, $id_shop = null)
+    public static function deleteAlert($id_customer, $customer_email, $id_product, $id_product_attribute, $id_shop, $notification_sent)
```

Comment:
is this a BC? 🤔

Reply:
Oh, maybe. I should have set `$id_shop = null, $notification_sent = true`.